angular.json

```diff
@@ -21,9 +21,10 @@
               "src/assets",
               "src/favicon.ico",
               {
-                "glob": "*",
-                "input": "node_modules/ngx-prx-styleguide/assets/images",
-                "output": "assets/images/"
+                "glob": "**/*",
+                "input": "node_modules/ngx-prx-styleguide/assets",
```

Comment:
I'm seeing 2 requests to `/assets/data/timezones.json` every time the field is rendered.  Should probably get that down to 1.

Also, this might be a prx-ng-serve problem, but that timezone file is set to `Cache-Control: no-cache`.  Which isn't ideal.

Reply:
All of our assets seem to be served with a `Cache-Control: no-cache` header, not just this particular file. I'll open a ticket in prx-ng-serve.

Reply:
See PRX/prx-ng-serve#4